src/languages/llvm.js

```diff
@@ -49,48 +96,28 @@ export default function(hljs) {
       'extractvalue insertvalue atomicrmw cmpxchg fence ' +
       'argmemonly double',
     contains: [
-      {
-        className: 'keyword',
-        begin: 'i\\d+'
-      },
-      hljs.COMMENT(
-        ';', '\\n', {relevance: 0}
+      TYPE,
+      hljs.COMMENT(/;\s*$/,
+        // this matches "empty comments"...
+        // ...because it's far more likely this is a statement terminator in
+        // another language than an actual comment
+        { relevance: 0 }
       ),
```

Comment:
I think this regex might be a bit too greedy?

![image](https://user-images.githubusercontent.com/1246453/99038054-c5649900-2539-11eb-9e69-15a1697e8187.png)

GitHub highlighting for reference

```llvm
;
;; foooo
define i32 @mul_add(i32 %x, i32 %y, i32 %z) {
  entry:
    %tmp = mul i32 %x, %y
    %tmp2 = add i32 %tmp, %z
    %tmp3 = add i32 %tmp, 0
    ret i32 %tmp3
}
```

Reply:
Better?